include/z64cutscene_commands.h

```diff
@@ -11,445 +11,273 @@
- * FORMAT
- *   Capital U is Unused
- *   qqqqssss eeeeUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU 00000000 00000000 00000000
- *   size = 0x30
+ * Starts a sequence at the specified time.
+ * @note The sequence ID is subtracted by 1 before being used. Add +1 to the desired sequence ID when passing it in.
  */
-#define CS_PLAY_BGM(sequence, startFrame, endFrame, unused0, unused1, unused2, unused3, unused4, unused5, unused6, unused7) \
-    CMD_HH(sequence, startFrame), CMD_HH(endFrame, unused0), \
-    CMD_W(unused1), CMD_W(unused2), CMD_W(unused3), CMD_W(unused4), CMD_W(unused5), \
-    CMD_W(unused6), CMD_W(unused7), 0x00000000, 0x00000000, 0x00000000
+#define CS_START_SEQ(seqId, frame, unused0, unused1, unused2, unused3, unused4, unused5, unused6, unused7, unused8) \
+    CMD_HH((seqId + 1), frame), CMD_HH(unused0, unused1), \
+    CMD_W(unused2), CMD_W(unused3), CMD_W(unused4), CMD_W(unused5), CMD_W(unused6), \
+    CMD_W(unused7), CMD_W(unused8), 0x00000000, 0x00000000, 0x00000000
 
 /**
- * ARGS
- *   s32 entries (e)
- * FORMAT
- *   00000057 eeeeeeee
- *   size = 0x8
+ * Declares a list of `CS_STOP_SEQ` entries.
  */
-#define CS_STOP_BGM_LIST(entries) CS_CMD_STOPBGM, CMD_W(entries)
+#define CS_STOP_SEQ_LIST(entries) \
+    CS_CMD_STOP_SEQ, CMD_W(entries)
 
 /**
- * ARGS
- *   s16 sequence (q), s16 startFrame (s), s16 endFrame (e)
- * FORMAT
- *   Capital U is Unused
- *   uuqqssss eeeeUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU 00000000 00000000 00000000
- *   size = 0x30
+ * Stops a sequence at the specified time.
+ * @note The sequence ID is subtracted by 1 before being used. Add +1 to the desired sequence ID when passing it in.
  */
-#define CS_STOP_BGM(sequence, startFrame, endFrame, unused0, unused1, unused2, unused3, unused4, unused5, unused6, unused7) \
-    CMD_HH(sequence, startFrame), CMD_HH(endFrame, unused0), \
-    CMD_W(unused1), CMD_W(unused2), CMD_W(unused3), CMD_W(unused4), CMD_W(unused5), \
-    CMD_W(unused6), CMD_W(unused7), 0x00000000, 0x00000000, 0x00000000
+#define CS_STOP_SEQ(seqId, frame, unused0, unused1, unused2, unused3, unused4, unused5, unused6, unused7, unused8) \
+    CMD_HH((seqId + 1), frame), CMD_HH(unused0, unused1), \
+    CMD_W(unused2), CMD_W(unused3), CMD_W(unused4), CMD_W(unused5), CMD_W(unused6), \
+    CMD_W(unused7), CMD_W(unused8), 0x00000000, 0x00000000, 0x00000000
 
 /**
- * ARGS
- *   s32 entries (e)
- * FORMAT
- *   0000007C eeeeeeee
- *   size = 0x8
+ * Declares a list of `CS_FADE_OUT_SEQ` entries.
  */
-#define CS_FADE_BGM_LIST(entries) CS_CMD_FADEBGM, CMD_W(entries)
+#define CS_FADE_OUT_SEQ_LIST(entries) \
+    CS_CMD_FADE_OUT_SEQ, CMD_W(entries)
 
 /**
- * ARGS
- *   s16 fadeType (t), s16 startFrame (s), s16 endFrame (e)
- * FORMAT
- *   Capital U is Unused
- *   ttttssss eeeeUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU UUUUUUUU 00000000 00000000 00000000
- *   size = 0x30
+ * Fades a sequence out over the specified duration.
```

Comment:
```suggestion
 * Fade out the sequence playing on the specified sequence player, over the specified frame range.
 * @see `CutsceneFadeOutSeqPlayer`
```
or something like that